### PR TITLE
Update Get started section of Fundamentals TOC

### LINKED ISSUE #20725: Update Core get started for .NET 5
core/get-started.md --> Hello World
standard/get-started --> Link to get started tutorials for each app type and get-started docs for each language.
 
---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 1c922021-b12a-83e4-3ee6-eceecd6076d2
* Version Independent ID: fdb655a4-cff4-6c67-7244-673760379663
* Content: [Get started with .NET Core](https://docs.microsoft.com/en-us/dotnet/core/get-started?tabs=windows)
* Content Source: [docs/core/get-started.md](https://github.com/dotnet/docs/blob/master/docs/core/get-started.md)
* Product: **dotnet-core**
* GitHub Login: @adegeo
* Microsoft Alias: **adegeo**

### PR DESCRIPTION
**Pull request by tdykstra**
Fixes #20725

### CODE REVIEW

File: docs/core/get-started.md [hunk 1]
```diff
@@ -23,57 +23,14 @@ dotnet new console --output sample1
...
-```console
+```output
 Hello World!
 ```
 
-Congratulations! You've created a simple .NET Core application. You can also use [Visual Studio Code](./tutorials/with-visual-studio-code.md), [Visual Studio](./tutorials/with-visual-studio.md) (Windows only), or [Visual Studio for Mac](tutorials/with-visual-studio-mac.md) (macOS only), to create a .NET Core application.
+Congratulations! You've created a simple .NET application.
 
-## Tutorials
+## Next Steps
```
Comment (IEvangelist) [inline docs/core/get-started.md:34]:
```suggestion
## Next steps
```

**Review by IEvangelist — COMMENTED** [review]
Sentence casing.

File: docs/core/get-started.md [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
 ms.author: adegeo
-ms.date: 12/03/2019
+ms.date: 09/29/2020
 ms.custom: vs-dotnet
 ---
-# Get started with .NET Core
+# Get started with .NET
 
-This article provides information on getting started with .NET Core. .NET Core can be installed on Windows, Linux, and macOS. You can code in your favorite text editor and produce cross-platform libraries and applications.
+This article tells you how to create and run a "Hello World!" .NET app.
```
Comment (gewarren) [inline docs/core/get-started.md:11]:
```suggestion
This article teaches you how to create and run a "Hello World!" .NET app.
```

**Review by gewarren — APPROVED** [review]
LGTM!

**Comment by tdykstra** [comment]
Thanks for the reviews, @gewarren and @IEvangelist ! 
@BillWagner if you get a chance to review I'll make a new PR to address your feedback.

**Comment by BillWagner** [comment]
@tdykstra I had looked, and I didn't have any additional comments. This is great.

**Pull request merged**